templates/beanstalk-docker.yaml

```diff
@@ -0,0 +1,94 @@
+---
+    AWSTemplateFormatVersion: '2010-09-09'
+    Description: AWS CloudFormation iAtlas API Test
+    Parameters:
+      Subnets:
+        Type: CommaDelimitedList
+        Description: Subnets
+      VpcId:
+        Type: String
+        Description: VPC Id
+      S3Bucket:
+        Type: String
+        Description: S3 Bucket name
+      S3Key:
+        Type: String
+        Description: S3 Key name
+      HostKeyName:
+        Type: String
+        Description: The host SSH key to assign to instances
+      FlaskEnvironment:
+          Type: String
+          Description: The Flask configuration key to pass to the app
+          ConstraintDescription: Acceptable keys for Flask configuration
+          AllowedValues:
+            - staging
+            - production
+    Resources:
+      iAtlasApiTest:
+        Type: AWS::ElasticBeanstalk::Application
+        Properties:
+          Description: iAtlas API Deployment
+
+      iAtlasApiVersion:
+        Type: AWS::ElasticBeanstalk::ApplicationVersion
+        Properties:
+          ApplicationName:
+            Ref: "iAtlasApiTest"
+          Description: Latest version
+          SourceBundle:
+            S3Bucket:
+              Ref: S3Bucket
```

Comment:
You need to create this bucket with cloudformation template as well.  You can do it in a separate template or in here.